src/components/Characters/Fox/__tests__/Fox.test.js

```diff
@@ -1,28 +1,28 @@
 import React from 'react';
 import { create } from 'react-test-renderer';
 
-import Fox from '../Fox';
+import Fox from '../index';
```

Comment:
I think it is better to actually import from precise location because some components are / could be wrapped in HoCs in **index.js**, which alters their functionality / accepted props.

Reply:
Fixed in c6073fe803add232415561bcb8753e478eb778bb